dev/prepare_db_gaps.pql

```diff
@@ -0,0 +1,78 @@
+// This is a Preql file, used for setting up a database for developement and testing
```

Comment:
Can we include this one in the other file? Or you think this is such an edge-case it's not really worth populating in the common dev environment?

Reply:
Yep, I think it's only worth testing if you mess with the basic algorithm, and most devs won't have anything to do with it.